scripts/job_manager.ps1

```diff
@@ -0,0 +1,94 @@
+class MyJobManager {
+    [String] $Name = "JobManager"
+    [Hashtable] $Jobs = @{}
+
+    JobManager () {
+    }
+
+    [void] AddJob ($Tracker, $ScriptBlock, $ArgumentList, $InitScript) {
+        $job = Start-Job -ScriptBlock $ScriptBlock `
+                         -ArgumentList $ArgumentList `
+                         -InitializationScript $InitScript
+        Start-Sleep -Milliseconds 100
```

Comment:
why do you need this sleep here?

Reply:
when the job starts, it is not in the Running state immediately.